pmlc/dialect/stripe/into_mlir.cc

```diff
@@ -474,6 +474,10 @@ static mlir::FuncOp ProgramIntoMLIR(MLIRContext* ctx, const stripe::Block& block
     func.setArgAttr(argIndex, attrName, builder.getStringAttr(ref.into()));
     auto attrLayout = Dialect::getDialectAttrName("layout");
     func.setArgAttr(argIndex, attrLayout, builder.getTypeAttr(tensorTypes[argIndex]));
+    auto attrs = TagsToDict(&builder, ref);
+    if (attrs.size()) {
+      func.setArgAttr(argIndex, Dialect::getDialectAttrName(Dialect::getStripeAttrsName()), attrs);
```

Comment:
If we're going to prefix the attr name with the dialect name, then I don't think we need to duplicate the prefix in the name. So instead of `stripe.stripe_attrs`, how about we go with just `stripe.attrs`

Reply:
SGTM.  (FYI, I'm also fixing the lit test -- forgot to actually test //pmlc :-)